### DISCUSSION TITLE
Override keys when focusing a certain app

### DISCUSSION THREAD

**Discussion by pitoniak32** [nikitabobko/AeroSpace#767] [questions-and-answers]
I tried to search previous discussions before asking, hopefully I didn't miss one.

But I would like to disable the `alt-{h,j,k,l}` keys when in Alacritty because I typically am using zellij for multiplexing terminals. Those keybinds are used in zellij for navigation. So I would like to just have AeroSpace pass them through and do nothing since I will more than likely have the alacritty window in its own space and can navigate to and from with `alt-#`

Thank you in advance!

**Comment by Dimentium** [comment]
> I tried to search previous discussions before asking, hopefully I didn't miss one.
> 
> 
> 
> But I would like to disable the `alt-{h,j,k,l}` keys when in Alacritty because I typically am using zellij for multiplexing terminals. Those keybinds are used in zellij for navigation. So I would like to just have AeroSpace pass them through and do nothing since I will more than likely have the alacritty window in its own space and can navigate to and from with `alt-#`
> 
> 
> 
> Thank you in advance!

I will assume that complex keyboard shortcut configurations make sense to do with specially designed programs for that purpose, such as Karabiner. 
A similar (from my pov) topic was touched upon in this discussion: https://github.com/nikitabobko/AeroSpace/discussions/640#discussioncomment-11198873
